keep-only-warm-data/README.md

```diff
@@ -0,0 +1,125 @@
+# Keep only warm data into Kuzzle
+
+## Requirements
```

Comment:
I personally prefer "Requirements" but I agree we can remove elasticssearch and redis since that is comprised in "Kuzzle X"

I don't know if "Since" is standard practice.

Reply:
I agree that Elasticsearch and Redis requirements could be skipped since they are part of Kuzzle requirements.  

For the format maybe we can wait to see how we integrate the How-To within the actual documentation (and also what we will be using, same template or not, etc..)

Reply:
@appacea > if `Since` is not standard practice, then we may want to change that tag in our documentation too. This is just a tag to edit in metalsmith, it's really quick to fix.
What about `Added in: x.y.z`, like in the Node.js documentation?

@Aschen > I didn't mean to reproduce the exact format, especially since we do not use metalsmith nor custom CSS here. Instead, what I have in mind is to have something somewhat homogeneous with the rest of our documentation.

Reply:
Despite I like `Added in x.y.z` for features in the documentation, I think this `Since Kuzzle 1.2.11` is more suitable for a How-To